### ISSUE TITLE
issue

### ISSUE THREAD

**Issue by Kiramatzu** [GaiolaDavid/Local-Alarm-for-Eve-Online#3]
Sir any idea why is detecting it like a keylogger?

**Comment by GaiolaDavid** [comment]
The program doesn't connect to the internet. It captures pictures on the region of a screen, analyzes them for specific icons, and makes a sound. Those are it's only functions.

**Comment by Kiramatzu** [comment]
yes i know what he do but i need to disable active scan passive scan and cloud scan for he to be able to run because is detecting it as a keylogger i dont why  , posibly part of the code you capture to create it was used before in some type of attack i check the hashing and is doing positive https://www.virustotal.com/gui/file/51436689c94f020f3479cd68551c148a8cfc0f6e382c149b56d1e1f3fd025af9/relations
